subcommands/git/cmd.go

```diff
@@ -73,8 +74,16 @@ func doGitCreds(cmd *cobra.Command, args []string) {
 	helperName := "fio"
 	dst := filepath.Join(helperPath, GIT_CREDS_HELPER)
 	if runtime.GOOS == "windows" {
-		dst += ".exe"
-		helperName += ".exe"
+		/*
+		** On Windows should use the absolute path.
+		** If you install git for windows, there is an option to not support symlinks everywhere.
+		** For some reason it will not run the configured credential helper command correctly.
+		** When using an absolute path the `\` should be replaced for `/` characters.
+		** In addition to that, spaces ` ` should be encoded to `\\ `.
+		** So for example the following path will be used: C:/Program\\ Files/Git/bin/git-credential-fio.exe
+		 */
```

Comment:
Once done, try to truncate this comment to at most 2 lines.

Reply:
nitpick - but this is an unusual way to do comments in golang (the two asterisk lines). https://tip.golang.org/doc/comment

Reply:
Fixed this.